### ISSUE TITLE
Screen sharing with audio doesn't ignore jitsi's output

### ISSUE THREAD

**Issue by AnasDevO** [jitsi/jitsi-meet-electron#988]
## Description
As title suggest, when ScreenSharing with audio enabled, other participants in the call will hear themselves in the audio, since jitsi fails to isolate it's own output.

## Current behavior
Jitsi output is shared with the audio streamed to the ScreenShare

## Expected Behavior
Jitsi should not stream other participant's audio stream into its own ScreenShare Audio stream

## Possible Solution
No idea.

## Steps to reproduce
Get in a call with two participants, both with microphone activated, and enable ScreenSharing with audio.
Once someone talks, there should be an Echo-like audio feedback. 

# Environment details
- Windows 10
- Intel CPU + Nvidia GPU if it matters.

**Comment by saghul** [comment]
@jallamsetty1 I think this is also another reason to move to gDM in Electron, IIRC that code path does enable the echo canceller but the gUM path doesn't, and patches to fix that have been rejected.

**Comment by jallamsetty1** [comment]
> @jallamsetty1 I think this is also another reason to move to gDM in Electron, IIRC that code path does enable the echo canceller but the gUM path doesn't, and patches to fix that have been rejected.

Yes! I will scheduling this work as part of the new LJM APIs for track creation/addition.

**Comment by AnasDevO** [comment]
Any update for this?

**Comment by AnasDevO** [comment]
Happy to hear that!
